go.sum

```diff
@@ -250,6 +250,8 @@ github.com/codahale/hdrhistogram v0.0.0-20160425231609-f8ad88b59a58/go.mod h1:sE
 github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd/go.mod h1:sE/e/2PUdi/liOCUjSTXgM1o87ZssimdTWN964YiIeI=
 github.com/compose-spec/compose-go v0.0.0-20210702130122-154903ab827c h1:mpdaqKEsLNPXKu97TRSDs6CMQy7kSHHYf7AJCVaGH6g=
 github.com/compose-spec/compose-go v0.0.0-20210702130122-154903ab827c/go.mod h1:5V65rPnTvvQagtoMxTneJ2QicLq6ZRQQ7fOgPN226fo=
+github.com/compose-spec/compose-go v0.0.0-20210706130854-69459d4976b5 h1:PpI72CT1bcVPNZyqI1HI8UhQVRVtqLb2tdwi5WphN3c=
+github.com/compose-spec/compose-go v0.0.0-20210706130854-69459d4976b5/go.mod h1:5V65rPnTvvQagtoMxTneJ2QicLq6ZRQQ7fOgPN226fo=
```

Comment:
This is odd that it doesn’t replace the previous one, isn’t it?

Reply:
Yep! I just ran a `go mod tidy` and it removed the old ones.